Documentation/contributing.rst

```diff
@@ -347,7 +347,7 @@ Introduction
 ~~~~~~~~~~~~
 
 Cilium uses `Ginkgo <https://onsi.github.io/ginkgo>`_ as a testing framework for
-writing end-to-end tests which test Cilium all the way from the API level (e.g. 
+writing end-to-end tests which test Cilium all the way from the API level (e.g.
```

Comment:
`writing end-to-end tests of Cilium all the way from` if you didn't want to use `test` so many times